### PR TITLE
Isolates support

### PR DESCRIPTION
**Pull request by felixblaschke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -483,6 +483,7 @@ Shelf Run uses a default configuration, that can be modified via **environment v
 | SHELF_PORT           | 8080          | Port to bind the shelf application to    |
 | SHELF_ADDRESS        | localhost     | Address to bind the shelf application to |
 | SHELF_HOTRELOAD      | true          | Enable hot-reload                        |
+| SHELF_SHARED_HTTP_SERVER      | false          | Shares the combination of port address between isolates                        |
```
Comment (felixblaschke) [inline README.md:486]: fix name

**Comment by felixblaschke** [comment]
Hey @davidmartos96, do you mind reviewing this? Are there any issues I overlooked?

I basically refined your PR and added a unit test.

**Comment by davidmartos96** [comment]
@felixblaschke LGTM, thank you!

**Comment by felixblaschke** [comment]
Alright... I published 1.2.0 with your isolates contributions: https://pub.dev/packages/shelf_plus/versions/1.2.0#multithreading

Thank you!

**Pull request merged**